ansible/roles/schulcloud-server-h5p-proxy/templates/api-h5p-proxy-ingress.yml.j2

```diff
@@ -29,13 +29,13 @@ spec:
 {% endif %}
 {% endif %}
   rules:
-  - host: {{ DOMAIN }}
+  - host: {{ H5P_PROXY_PREFIX |default("h5p-") }}{{ DOMAIN }}

```

Comment:
Don't set at every part where you use the H5P_PROXY_PREFIX  the default, it's better to add an h5p_proxy.yml to the ansible/group_vars/all/ and set there the default for all instances

Reply:
Agreed